include/SFML/Graphics/Texture.hpp

```diff
@@ -61,7 +61,7 @@ class SFML_GRAPHICS_API Texture : GlResource
         Pixels      //!< Texture coordinates in range [0 .. size]
     };
 
-public:
+
```

Comment:
```suggestion
```
I think I'd rather delete these extra `private:` tags than replace them with empty space. We don't need these extra empty newlines.

Reply:
Yeah, I'll go hunt for them. I thought that the `format` build might catch them, but apparently not.

Reply:
We allow for up to 2 consecutive empty newlines which is why this wasn't automatically removed.